cli/tools/repl/editor.rs

```diff
@@ -414,7 +414,9 @@ impl ReplEditor {
     let mut editor =
       Editor::with_config(editor_config).expect("Failed to create editor.");
     editor.set_helper(Some(helper));
-    editor.load_history(&history_file_path).unwrap_or(());
+    if let Some(history_file_path) = &history_file_path {
+      editor.load_history(history_file_path).unwrap_or(());
```

Comment:
**nitpick:** Isn't this essentially equivalent to doing `let _ = editor.load_history(history_file_path);`? (Just personal curiosity.)

Reply:
Yes, or simply `_ = editor.load_history(history_file_path);`, or even `editor.load_history(history_file_path).ok();`. I saw no reason to change it and left it as-is.